config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
   root to: "home#index"
+  post '/', to: 'home#results'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.